com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.LightLoop.cs

```diff
@@ -842,11 +842,11 @@ TextureHandle GenerateMaxZPass(RenderGraph renderGraph, HDCamera hdCamera, Textu
                 {
                     passData.parameters = PrepareGenerateMaxZParameters(hdCamera, depthMipInfo);
                     passData.depthTexture = builder.ReadTexture(depthTexture);
-                    passData.maxZ8xBuffer = builder.CreateTransientTexture(new TextureDesc(Vector2.one * 0.125f, false, true)
+                    passData.maxZ8xBuffer = builder.CreateTransientTexture(new TextureDesc(Vector2.one * 0.125f, true, true)
```

Comment:
So this is a tad trickier than this actually given that the VBuffer is not scaled; this change is fine but it will over estimate and potentially lead to worse performance. 

So for the bug fix sake this is fine, but I'll keep it in mind as we might need to change it a bit.

Reply:
@kecho can you add a comment there (TODO) so we don't forget, thanks